upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -603,6 +615,117 @@ func (c *ApplyClusterCmd) upgradeSpecs() error {
 	return nil
 }
 
+// validateKopsVersion ensures that kops meet the version requirements / recommendations in the channel
+func (c *ApplyClusterCmd) validateKopsVersion(channel *api.Channel) error {
+	kopsVersion, err := semver.Parse(kops.Version)
+	if err != nil {
+		glog.Warningf("unable to parse kops version %q", kops.Version)
+		// Not a hard-error
+		return nil
+	}
+
+	versionInfo := api.FindVersionInfo(channel.Spec.KopsVersions, kopsVersion)
+	if versionInfo == nil {
+		glog.Warningf("unable to find version information for kops version %q in channel", kopsVersion)
+		// Not a hard-error
+		return nil
+	}
+
+	recommended, err := api.FindRecommendedUpgrade(versionInfo, kopsVersion)
+	if err != nil {
+		glog.Warningf("unable to parse version recommendation for kops version %q in channel", kopsVersion)
+	}
+
+	required, err := api.IsUpgradeRequired(versionInfo, kopsVersion)
+	if err != nil {
+		glog.Warningf("unable to parse version requirement for kops version %q in channel", kopsVersion)
+	}
+
+	if recommended != "" && !required {
+		fmt.Printf("\n")
+		fmt.Printf("*******************************************************************\n")
+		fmt.Printf("A new kops version is available: %s\n", recommended)
+		fmt.Printf("Upgrading is recommended\n")
```

Comment:
Can we point to something here for reference? `https://github.com/kubernetes/kops/UPGRADE.md` for instance.. so we can have some documentation (that can be easily changed) for the user?

We can just point it to the installation section for day 1 purposes..